XenModel/FriendlyNames.Designer.cs

```diff
@@ -4626,7 +4626,7 @@ public class FriendlyNames {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Pool master changed.
+        ///   Looks up a localized string similar to Pool coordinator changed.
         /// </summary>
         public static string Message_name_pool_master_transition {

```

Comment:
master

Reply:
Here wasn't sure if the name was relevant to anything in particular as well. Maybe you or @kc284 can advise on if we want to change this instance, as it seems to mirror the camel case of the API. However I could not find anything on the OSS docs so maybe this is just a coincidence.

Reply:
In most cases where we retrieve a Friendly name by key, the latter is a string coming as is from the server, so we shouldn't change the keys in FriendlyNames, only their values.